### DISCUSSION TITLE
Is there a way to get data filtered data from the table?

### DISCUSSION THREAD

**Discussion by ryanmhorstman** [TanStack/table#4729] [Q&A] [answered]
I am looking to retrieve the current state of the data showing in the table, after applying filters. I notice you can access the data by: `table.options.data` but that is the entire data set not the filtered data. I've scoured the docs and cannot find an answer. Thank you!

**Comment by ryanmhorstman** [comment, marked as answer]
I took a look with fresh eyes today and was able to get the current state of the table with the following method:

`table.getSortedRowModel().rows.map(row => row.original)`

**Comment by KevinVandy** [comment]
`table.getFilteredRowModel().rows.map(row => row.original)`
